src/storage/wal_replay.cpp

```diff
@@ -427,11 +427,10 @@ void ReplayWithoutIndex(ClientContext &context, Catalog &catalog, AlterInfo &inf
 	catalog.Alter(context, info);
 }
 
-void ReplayIndexData(AttachedDatabase &db, BinaryDeserializer &deserializer, IndexStorageInfo &info,
-                     const bool deserialize_only) {
+void ReplayIndexData(ClientContext &context, StorageManager &storage_manager, BinaryDeserializer &deserializer,
```

Comment:
Maybe this function can be moved into the `WriteAheadLogDeserializer` as a private function